WordPress/Classes/ViewRelated/Me/MyProfileViewController.swift

```diff
@@ -7,19 +7,52 @@ func MyProfileViewController(account: WPAccount) -> ImmuTableViewController? {
     }
 
     let service = AccountSettingsService(userID: account.userID.intValue, api: api)
-    return MyProfileViewController(service: service)
+    let headerView = MakeHeaderView(account: account)
+    return MyProfileViewController(account: account, service: service, headerView: headerView)
 }
 
-func MyProfileViewController(service: AccountSettingsService) -> ImmuTableViewController {
-    let controller = MyProfileController(service: service)
+func MyProfileViewController(account: WPAccount, service: AccountSettingsService, headerView: MyProfileHeaderView) -> ImmuTableViewController {
+    let controller = MyProfileController(account: account, service: service, headerView: headerView)
     let viewController = ImmuTableViewController(controller: controller)
+    headerView.onAddUpdatePhoto = {
+        controller.presentGravatarPicker(viewController)
+    }
+    viewController.tableView.tableHeaderView = headerView
     return viewController
 }
 
+func MakeHeaderView(account: WPAccount) -> MyProfileHeaderView {
```

Comment:
I'll take the blame for this since I opened the door with the above `MyProfileViewController`, which was meant to look like an initializer for a nonexistent `MyProfileViewController` type. That was hacky and I have lots of regrets already about that code 😬

I think this method could live inside `MyProfileViewController`, but also methods should generally start with lowercase. Top level functions are not very common in our codebase, although I'd personally be fine with more of them, as long as they're `private`.

Reply:
I'm not sure how I would move that method inside of `MyProfileViewController` unless I also pass in the reference to the ImmuTableViewController, `viewController`, so I can attach the `tableHeaderView`. I was trying to avoid that because it looks like the intention of the class was to not manipulate the viewController except by using delegate methods. 

Made `private` and added lowercase method name.